programming-ide/working-shortcuts.md

```diff
@@ -16,81 +22,64 @@ The following keyboard shortcuts can be used in a variety of ways. If you forget
 
 ### Moving the Current Line
 
-On a Mac, press `option+up` or `option+down` to move the current line up or down. 
+Press `alt+up` or `alt+down` to move the current line up or down. 
 
-On other computers, press `alt+up` or `alt+down`.
 
 ![move line](/images/repls/editor/move_line.gif)
 
 ### Clone the Current Line
 
-On a Mac, press `shift+option+up` or `shift+option+down` to clone the current line above or below.
+Press `shift+alt+up` or `shift+alt+down` to clone the current line above or below.
 
-On other computers, press `shift+alt+up` or `shift+alt+down`.
 
 ![clone line](/images/repls/editor/clone_line.gif)
 
 ### Selecting Lines
 
-On a Mac, press `cmd+I` to select a line. 
+Press `ctrl+L` to select a line. 
 
-On other computers, press `ctrl+I`.
 
 ![select line](/images/repls/editor/select_line.gif)
 
 ### Deleting Lines
 
-On a Mac, press `shift+cmd+K` to delete a line.
+Press `ctrl+shift+K` to delete a line.
 
-On other computers, press `ctrl+shift+K`.
 
 ![delete line](/images/repls/editor/delete_line.gif)
 
 ### Inserting Lines
 
-On a Mac, press `shift+cmd+enter` to insert a line above.
+Press `ctrl+shift+enter` to insert a line above.
 
-On other computers, press `ctrl+shift+enter`.
 
 ![insert line](/images/repls/editor/insert_line.gif)
 
 ### Adding Cursors
 
-On a Mac, press `cmd+option+up` or `cmd+option+down` to add cursors above or below your current line. 
+Press `ctrl+alt` or `ctrl+alt` and then click with the mouse to add more cursors. 
```

Comment:
@jamesdanielwhitford - does pressing ctrl+alt do anything on its own? For me (on mac) it only works if I also click. I'd also change "click with the mouse" to "left click" just to make it unambiguous.

Reply:
@sixhobbits Yes that makes sense. It should say `ctrl+alt+left click`